packages/moldova/README.md

```diff
@@ -0,0 +1,61 @@
+# 🇲🇩 Moldova
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the districts of Moldova:
+
+- Anenii Noi
+- Bălţi
+- Tighina
+- Briceni
+- Basarabeasca
+- Cahul
+- Călăraşi
+- Cimişlia
+- Criuleni
+- Căuşeni
+- Cantemir
+- Chişinău
+- Donduşeni
+- Drochia
+- Dubăsari
+- Edineţ
+- Făleşti
+- Floreşti
+- Unitatea Teritorială Autonomă Găgăuzia (UTAG)
+- Glodeni
+- Hînceşti
+- Ialoveni
+- Leova
+- Nisporeni
+- Ocniţa
+- Orhei
+- Rezina
+- Rîşcani
+- Şoldăneşti
+- Sîngerei
+- Unitatea Teritorială din Stînga Nistrului
+- Soroca
+- Străşeni
+- Ştefan Vodă
+- Taraclia
+- Teleneşti
+- Ungheni
```

Comment:
Is it an up-to-date map?
According to [wikipedia](https://en.wikipedia.org/wiki/Administrative_divisions_of_Moldova) these are actually districts, municipalities and autonomous territories, but it doesn't contain [Transnistria](https://en.wikipedia.org/wiki/Transnistria_autonomous_territorial_unit), and [Tighina](https://en.wikipedia.org/wiki/Tighina_County_(Moldova)) is an old county.

Reply:
@VictorCazanave  I used the original svg maps; you are right about Tighina - this is the old name of the county, so I replaced it with the current name - Bender.
Transnistria is already listed on the map, except officially (in the docs) the Moldovan state calls it ”Unitatea Teritorială din Stânga Nistrului”. It is more of a political statement, as Transnistria is not recognised as a state; similarly to ”UTA Găgăuză” I extended the label and simply added ”(Transnistria)” in brackets. 
Please review